examples/raytracing/example_raytracing.cpp

```diff
@@ -9,81 +9,188 @@
  * SPDX-License-Identifier: BSD-3-Clause                                    *
  ****************************************************************************/
 
+/*
+ * This example demonstrates how to use ArborX for a raytracing example where
+ * rays carry energy that they deposit onto given boxes as they hit them.
+ * The order in which these rays (which originate from one of the boxes) hit
+ * the boxes is important in this case, since the ray loses energy on
+ * intersection. The example shows two different ways to do that:
+ * 1.) using a specialized traversal that orders all intersection in a heap
+ *     so that the callbacks for a specific ray are called in the correct order
+ *     (OrderedIntersectsBased namespace).
+ * 2.) storing all intersections and doing the deposition of energy in a
+ *     postprocessing step (IntersectsBased namespace).
+ */
+
 #include <ArborX.hpp>
 #include <ArborX_Ray.hpp>
-#include <ArborX_Version.hpp>
 
-#include <Kokkos_Core.hpp>
```

Comment:
Why did you remove?

Reply:
I just replaced the example. I'm not sure how valuable it is to look at the diff.